integration_tests/seeds/sf_event_history_data.csv

```diff
@@ -0,0 +1,11 @@
+_fivetran_active,_fivetran_start,_fivetran_end,id,_fivetran_synced,account_id,activity_date,activity_date_time,created_by_id,created_date,description,end_date_time,event_subtype,group_event_type,invitee_uuid_c,is_child,is_deleted,is_group_event,is_recurrence,is_reminder_set,last_modified_by_id,last_modified_date,location,no_show_c,owner_id,start_date_time,subject,type,what_count,what_id,who_count,who_id,end_date,is_archived
```

Comment:
Let's remove this since we are not using it in the end models anymore.

Reply:
Removed